.github/workflows/release_notes.yml

```
@@ -0,0 +1,69 @@
+name: Generate Release Notes
+
+on:
+  pull_request:
+
+# on:
+#   push:
+#     branches:
+#       - main
+
+jobs:
+  send-pull-requests:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: 3.1
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Generate release notes and environment variables
+        run: |
+          ruby -r /home/runner/work/newrelic-ruby-agent/newrelic-ruby-agent/.github/workflows/scripts/generate_release_notes.rb -e GenerateReleaseNotes.new.write_output_file
+          echo "file_name=$(ruby -r /home/runner/work/newrelic-ruby-agent/newrelic-ruby-agent/.github/workflows/scripts/generate_release_notes.rb -e GenerateReleaseNotes.new.file_name)" >> $GITHUB_ENV
+          echo "branch_name=$(ruby -r /home/runner/work/newrelic-ruby-agent/newrelic-ruby-agent/.github/workflows/scripts/generate_release_notes.rb -e GenerateReleaseNotes.new.branch_name)" >> $GITHUB_ENV
+
+      - name: Build source file
+        run: echo "source_file=/home/runner/work/newrelic-ruby-agent/newrelic-ruby-agent/${{env.file_name}}" >> $GITHUB_ENV
+
+      - name: Create branch
+        uses: dmnemec/copy_file_to_another_repo_action@main
+        env:
+          API_TOKEN_GITHUB: ${{ secrets.RELEASE_NOTES }} # Need new token from NR open bot!
+        with:
+          source_file: "${{env.source_file}}"
+          destination_repo: 'hannahramadan/docs-website' # newrelic/docs-website
+          destination_folder: 'src/content/docs/release-notes/agent-release-notes/ruby-release-notes'
+          user_email: 'hannahr315@gmail.com' # nr-opensource-bot
+          user_name: 'hannahramadan' # opensource+bot@newrelic.com
+          destination_branch: 'develop' # dev
+          destination_branch_create: ${{env.branch_name}}
+          commit_message: 'chore(ruby agent): add release notes'
```

Should this have a space?

```suggestion
          commit_message: 'chore (ruby agent): add release notes'
```

---

It doesn't look like the docs team usually does spaces!